.github/ISSUE_TEMPLATE/missing-capability.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Missing capability
+about: Report missing playbook capabilities
+title: Missing - lacking feature
+labels: bug, enhancement
+assignees: ''
+
+---
+
+*NOTE: This is not a feature request! This is to report a blind-spot in the play-books that limits expected usage*
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Maybe reword as 'Is your request related to a problem' or similar (since explicitly stated it's not a feature request)?